composer.json

```diff
@@ -18,13 +18,18 @@
         {
             "type": "git",
             "url": "git@github.com:richardkorthuis/geoPHP.git"
+        },
+        {
+            "type": "git",
+            "url": "git@bitbucket.org:acato/openkaarten-base-functions.git"
```

Comment:
Deze repository staat in bitbucket op Private, dat is een no-go. Of de repo moet publiek gemaakt worden, of hij moet publiek in de OpenWebconcept github (dat laatste heeft denk ik de voorkeur)

Reply:
Ik heb de repo in de OWC github geplaatst en in de composer.json aangepast